material_studies/rally/.gitignore

```diff
@@ -0,0 +1,70 @@
+# Miscellaneous
```

Comment:
Do we need this? Is there not one at the top of this repo?

Reply:
I think there is one at the top, but it's not as comprehensive as this one. I took this from Shrine.